platform/ios/src/MGLMapView.mm

```diff
@@ -386,6 +386,7 @@ - (void)commonInit
 {
     _isTargetingInterfaceBuilder = NSProcessInfo.processInfo.mgl_isInterfaceBuilderDesignablesAgent;
     _opaque = NO;
+    _changeDelimiterSuppressionDepth = 0;
```

Comment:
FYI, this is OK but technically unecessary, since Objective-C zeroes out ivars when allocating an object.